configs/opencv-ovms/scripts/run_grpc_kserv_bit-predict.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Similar to the python files we can look into refactoring the scripts into a single script for different setups.

Reply:
Address in future PR